tests/unit/number.js

```diff
@@ -410,14 +427,14 @@ define([
 
 
 			// NumberRegression.Test4052223
-			checkParse({ pattern: '#,#00.00' }, 'abc3');
+			checkParse({ pattern: '#,#00.00' }, 'abc3', NaN);
 
 			//TODO: got NaN instead of 1.222, is it ok?
 			//checkParse({pattern:'#,##0.###',locale:'en-us'},'1.222,111',1.222);
 			//checkParse({pattern:'#,##0.###',locale:'en-us'},'1.222x111',1.222);
 
 			//got NaN for illegal input, ok
-			checkParse(null,'hello: ,.#$@^&**10x');
+			checkParse(null,'hello: ,.#$@^&**10x', NaN);
```

Comment:
Due to modification in the `checkParse()` function, the expected value is now required. Since `hello: ,.#$@^&**10x` is not a number, the expected value is `NaN`.